test/metabase/search/impl_test.clj

```diff
@@ -18,7 +18,7 @@
 
 (deftest ^:parallel parse-engine-test
   (testing "Default engine"
-    (is (= :search.engine/in-place (#'search.impl/parse-engine nil))))
```

Comment:
This test would otherwise fail for people with the experimental flag. Ultimately all we care about is that it resolves so *some* valid engine.